e2e-tests/tests/test_e2e_integration_flows.py

```diff
@@ -239,3 +243,38 @@ def test_demo_website_text_returns_correct_anonymized_version():
     # Assert equal
 
     assert expected_anonymized_text == actual_anonymized_text
+
+
+@pytest.mark.package
```

Comment:
pytest complains it doesn't know what this is

Reply:
fixed